app/src/main/java/com/framgia/englishconversation/screen/conversationdetail/ConversationDetailActivity.java

```diff
@@ -29,13 +32,42 @@ public static Intent getInstance(Context context, TimelineModel timelineModel) {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        ActivityConversationDetailBinding binding = DataBindingUtil
+         mBinding = DataBindingUtil
```

Comment:
ERROR: 'method def' child have incorrect indentation level 9, expected level should be 8.